OctaneVSPlugin/ViewModel/DetailedItemViewModel.cs

```diff
@@ -145,15 +148,18 @@ public DetailedItemViewModel(BaseEntity entity)
                     if (string.IsNullOrEmpty(imageName))
                         continue;
 
-                    downloadTasks.Add(Task.Run(() =>
+                    var imagePath = TempPath + EntityType + Entity.Id + imageName;
+                    if (!File.Exists(imagePath))
+                    {
+                        imageNodes.Add(new Tuple<Element, string>(image, imagePath));
+                        downloadTasks.Add(_octaneService.DownloadAttachmentAsync(relativeUrl, imagePath));
+                    }
+                    else
                     {
-                        var imagePath = TempPath + EntityType + Entity.Id + imageName;
-                        if (!File.Exists(imagePath))
-                        {
-                            _octaneService.DownloadAttachmentAsync(relativeUrl, imagePath).Wait();
-                        }
                         image.Attr("src", imagePath);
-                    }));
+                    }
+
+                    updatedImage = true;
```

Comment:
is this value accurate, wound't this be conditioned on the download task being completed and successful (for below)

Reply:
@atothhpe Actually the variable name was misleading